chicken-sloths/bangazon-cli-sprint2
# Mb add cust prod c

# Description
This PR covers:
-Product Types: tests and model for retrieving all product types
-Prompt for user to add new product (posting to db)

## Related Ticket(s)
#4 

## Problem to Solve
When a user wants to add a product, they are required to fill in the fields to post to db

## Proposed Changes
We should to discuss our tests more thoroughly soon. The specific restraints on exact values are beginning to be cumbersome with each PR as our data evolves. I do not think it should be one person's task with every PR to adjust the specific tests to make sure they are flexible enough to pass.

## Expected Behavior
A user can post a new product to sell to the db

## Steps to Test Solution

1. rebuild DB
2. run ```npm start```
3. select option 2 to select an active customer
4. select option 4 to add new product
5. enter appropriate responses to prompt
6. check db to ensure this was added

## Testing

- [x] I certify that all existing tests pass (ALSO SEE NOTES IN PROPOSED CHANGES)

## Documentation
